### ISSUE TITLE
`Env:-ar` and `:-kr` incorrectly use `mul` and `add` parameters

### ISSUE THREAD

**Issue by brianlheim** [supercollider/supercollider#2858]
(Taken directly from @jamshark70's comment in #2855.)

I just noticed that `Env:ar` and `:kr` don't use `mul` and `add` correctly.

Everywhere else, `mul` and `add` can be modulated at any time.

These methods map `mul` and `add` onto `levelScale` and `levelBias` -- meaning that changes to the control inputs are ignored until the next envelope segment begins. `mul` and `levelScale` are related but not interchangeable.

Perhaps change them like this?

```
	kr { arg doneAction = 0, gate = 1.0, timeScale = 1.0, mul = 1.0, add = 0.0;
		^EnvGen.kr(this, gate, 1, 1, timeScale, doneAction).madd(mul, add)
	}

	ar { arg doneAction = 0, gate = 1.0, timeScale = 1.0, mul = 1.0, add = 0.0;
		^EnvGen.ar(this, gate, 1, 1, timeScale, doneAction).madd(mul, add)
	}
```

I apologize that I hadn't noticed before... But, if the help is going to start recommending `Env(something).kr` then I think `mul` and `add` should behave as they do with other UGens.

----

I would rather have it go the other way, and rename `mul` and `add` to `levelScale` and `levelBias`, because that is (1) more transparent (doesn't also add a call to `madd`), (2) not a behavior change, and (3) doesn't cut these functions off from access to `levelScale` and `levelBias`, which is probably the desired effect if the purpose of these calls is to forward arguments to `EnvGen`.

**Comment by snappizz** [comment]
Inbuilt `mul` and `add` arguments for UGens are [unnecessary these days](http://new-supercollider-mailing-lists-forums-use-these.2681727.n2.nabble.com/Why-the-quot-add-quot-arg-td7630586.html), and the distinctions between mul/add and levelScale/levelBias are neither obvious nor useful.

Part of me wants to recommend removing them entirely, but objectively Brian's choice is probably best.

**Comment by jamshark70** [comment]
> Inbuilt mul and add arguments for UGens are unnecessary these days,

Agreed.

> and the distinctions between mul/add and levelScale/levelBias are neither obvious nor useful.

... until you need the non-modulating behavior of levelScale/levelBias, but (at some hypothetical point in the future) somebody had removed it because "the distinction... [isn't] useful."

Let's not get carried away here.

I can go along with Brian's idea rather than my own.

**Comment by brianlheim** [comment]
Fixed by #2866.
